pkgs/tools/security/sbomnix/default.nix

```diff
@@ -1,53 +1,84 @@
 { lib
 , fetchFromGitHub
-, coreutils
-, curl
-, gnugrep
-, gnused
-, gzip
+, grype
 , nix
+, nix-visualize
 , python
-  # python libs
+, vulnix
+, # python libs
+  beautifulsoup4
 , colorlog
+, dfdiskcache
 , graphviz
 , numpy
 , packageurl-python
+, packaging
 , pandas
+, pyrate-limiter
 , requests
+, requests-cache
+, requests-ratelimiter
 , reuse
+, setuptools
 , tabulate
+,
 }:
 
 python.pkgs.buildPythonApplication rec {
   pname = "sbomnix";
-  version = "1.4.5";
+  version = "1.6.1";
+  format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "tiiuae";
-    repo = pname;
+    repo = "sbomnix";
     rev = "refs/tags/v${version}";
-    hash = "sha256-RxDFxVGivVBw2uhtzf231Q2HHTBFKSqGrknr2Es/ygM=";
+    hash = "sha256-kPjCK9NEs3D0qFsSSVX6MYGKbwqeij0svTfzz5JC4qM=";
+
+    # Remove documentation as it contains references to nix store
+    postFetch = ''
+      rm -fr "$out"/doc
+      find "$out" -name '*.md' ! -name "README.md" -exec rm -f '{}' \;
+    '';
   };
 
-  makeWrapperArgs = [
-    "--prefix PATH : ${lib.makeBinPath [ coreutils curl gnugrep gnused gzip graphviz nix ]}"
-  ];
+  postInstall = ''
+    wrapProgram $out/bin/sbomnix \
+      --prefix PATH : ${lib.makeBinPath [nix graphviz]}
+    wrapProgram $out/bin/nixgraph \
+      --prefix PATH : ${lib.makeBinPath [nix graphviz]}
+    wrapProgram $out/bin/vulnxscan \
+      --prefix PATH : ${lib.makeBinPath [grype nix vulnix]}
+    wrapProgram $out/bin/nix_outdated \
+      --prefix PATH : ${lib.makeBinPath [nix-visualize]}
+    wrapProgram $out/bin/provenance \
+      --prefix PATH : ${lib.makeBinPath [nix]}
+  '';
+
+  nativeBuildInputs = [ setuptools ];
 
   propagatedBuildInputs = [
+    beautifulsoup4
     colorlog
+    dfdiskcache
     graphviz
     numpy
     packageurl-python
+    packaging
     pandas
+    pyrate-limiter
     requests
+    requests-cache
+    requests-ratelimiter
     reuse
     tabulate
   ];
 
   pythonImportsCheck = [ "sbomnix" ];
+  doCheck = false;
```

Comment:
What's with the tests here https://github.com/tiiuae/sbomnix/tree/main/tests ? You didn't leave a comment if they require network or so. Can we run them?

Reply:
Added a comment in https://github.com/NixOS/nixpkgs/commit/1dad22788d6b2c1a7f9354f98236455dfad5eabe